components/properties/src/props.rs

```diff
@@ -476,54 +476,53 @@ impl<T: TrieValue> PropertyEnumToValueNameLinearMapperBorrowed<'static, T> {
     }
 }
 
-/// Private marker type for PropertyEnumToValueNameLinearTiny4Mapper
+/// Private marker type for PropertyScriptToIcuScriptMapper
 /// to work for all properties at once
 #[derive(Clone, Copy, PartialEq, Eq, Hash, Debug)]
 pub(crate) struct ErasedEnumToValueNameLinearTiny4MapV1Marker;
 impl DynamicDataMarker for ErasedEnumToValueNameLinearTiny4MapV1Marker {
-    type DataStruct = PropertyEnumToValueNameLinearTiny4MapV1<'static>;
+    type DataStruct = PropertyScriptToIcuScriptMapV1<'static>;
 }
 
 /// A struct capable of looking up a property name from a value
 /// Access its data by calling [`Self::as_borrowed()`] and using the methods on
-/// [`PropertyEnumToValueNameLinearTiny4MapperBorrowed`].
+/// [`PropertyScriptToIcuScriptMapperBorrowed`].
 ///
-/// This mapper is used for properties with sequential values and names with four or fewer characters,
-/// like the [`Script`] short names.
-/// It may be obtained using methods like [`Script::get_enum_to_short_name_mapper()`].
+/// This mapper is used for the [`Script`] property.
+/// It may be obtained using methods like [`Script::get_enum_to_icu_script_mapper()`].
 ///
 /// # Example
 ///
 /// ```
 /// use icu::properties::Script;
-/// use tinystr::tinystr;
+/// use icu::locale::subtags::script;
 ///
-/// let lookup = Script::enum_to_short_name_mapper();
-/// assert_eq!(lookup.get(Script::Brahmi), Some(tinystr!(4, "Brah")));
-/// assert_eq!(lookup.get(Script::Hangul), Some(tinystr!(4, "Hang")));
+/// let lookup = Script::enum_to_icu_script_mapper();
+/// assert_eq!(lookup.get(Script::Brahmi), Some(script!("Brah")));
+/// assert_eq!(lookup.get(Script::Hangul), Some(script!("Hang")));
 /// ```
 #[derive(Debug)]
-pub struct PropertyEnumToValueNameLinearTiny4Mapper<T> {
+pub struct PropertyScriptToIcuScriptMapper<T> {
```

Comment:
thought: not sure if the external name should be changed, but not opposed

Reply:
well it's definitely not Tiny4 anymore

Reply:
yeah good point